### DISCUSSION TITLE
Feature: /is baltop

### DISCUSSION THREAD

**Discussion by Sulphinx** [BG-Software-LLC/SuperiorSkyblock2#1341] [Approved Feature Requests]
The title should give enough information.
However i think a lot of player's could use a /is baltop so they can see the progress they are making.
/is baltop showing top balances of islands from 1-10 and then make it possible to do /is baltop 2 to show page 2 of baltop.

**Comment by djtmk1** [comment]
Isn't that what /baltop with essentials is for? I am jw is all.

**Reply by Sulphinx** [reply]
Correct, but if you add money to your island balance, you can only view eachothers balances by typing /is bal (island) instead og /is baltop
Cause when you add money to your island balance, it removes it from your essentials balance, and adds it to your islands balance.

**Comment by Lemvs** [comment]
Added as new sorting type for /is top.

@OmerBenGera you can close it.
